spec/models/spree/order_contents_spec.rb

```diff
@@ -29,12 +29,12 @@ module Spree
         before do
           order.create_proposed_shipments
           order.touch :completed_at
+          order.shipments.first.save
```

Comment:
We can't expect people to explicitly save proposed shipments to have the same behavior as we did before. Maybe we need to backport the change @waiting-for-dev did on past Solidus versions that can work with Rails 6.1?

Reply:
Absolutely. I'll work on this.

Reply:
Removed these lines of code. Extension will fail until the backport is completed.